src/NewTwitchApi/Resources/GamesApi.php

```diff
@@ -25,4 +25,22 @@ public function getGames(array $ids = [], array $names = []): ResponseInterface
 
         return $this->callApi('games', $queryParamsMap);
     }
+
+    /**
+     * @param int $first The number of items to return, default is 20
```

Comment:
No need for any of the @param or @return tags. The descriptions are in Twitch's documentation, which can be found in the @link tag, and the types should be type hints in the method signature.